js/app.js

```diff
@@ -103,19 +103,60 @@ Event.on(document.body, 'ui:click', '.show-tutorial', function(evt){
 Event.on(document.body, 'ui:click', '.do-run', startScript);
 Event.on(document.body, 'ui:click','.do-stop', stopScript);
 Event.on('.do-pause', 'ui:click', null, function () {
-    /* TODO */
+    console.assert(!!process, 'Trying to pause a process that DOES NOT EXIST');
+
+    if (!process) {
+        console.warn('Clicked pause when there is no process running.');
+        return;
+    }
+    process.pause();
 });
-Event.on('.do-step', 'ui:click', null, function () {
-    /* TODO */
+Event.on('.do-step', 'ui:click', null, function (evt) {
+    if (!process) {
+        /* FIXME: when loading a new script... this process should be
```

Comment:
For FIXMEs, if you aren't going to get to it, make sure you add an issue to the tracker.